From d469efb9c9825f20baea36e995590fc3e3b4f074 Mon Sep 17 00:00:00 2001
From: Grace Cham <hscham@chromium.org>
Date: Thu, 28 Jul 2022 13:04:39 +0900
Subject: [PATCH] backward compatibility: add SharedMemoryHandle back into
 PlatformSharedMemoryRegion

Alias base::subtle::PlatformSharedMemoryRegion::ScopedPlatformHandle as
base::subtle::ScopedPlatformSharedMemoryHandle since it is moved out of
PlatformSharedMemoryRegion as its own type in r991946
(crrev.com/c/3574675).

Change-Id: Iab81130a444f3cd670f4a4edd59f49e314ec7766
---
 base/memory/platform_shared_memory_region.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/base/memory/platform_shared_memory_region.h b/base/memory/platform_shared_memory_region.h
index 74d1effab2..58520232d3 100644
--- a/base/memory/platform_shared_memory_region.h
+++ b/base/memory/platform_shared_memory_region.h
@@ -103,6 +103,10 @@ class BASE_EXPORT PlatformSharedMemoryRegion {
   };
 #endif
 
+  using ScopedPlatformHandle
+      [[deprecated("Use base::subtle::ScopedPlatformSharedMemoryHandle.")]] =
+          base::subtle::ScopedPlatformSharedMemoryHandle;
+
   // The minimum alignment in bytes that any mapped address produced by Map()
   // and MapAt() is guaranteed to have.
   enum { kMapMinimumAlignment = 32 };
-- 
2.37.1.595.g718a3a8f04-goog

